src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -14419,23 +14419,20 @@ class DeleteShelfAsyncFlattenedAsyncEmptyResponseTypeWithResponseHandling
    /// <summary>
    /// Test response handling for methods that return empty
    /// </summary>
-   public static void SampleDeleteShelf()
+   public static async Task SampleDeleteShelfAsync()
    {
-       public static async Task SampleDeleteShelfAsync()
-       {
-           LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-           ShelfName name = new ShelfName("[SHELF_ID]");
-           libraryServiceClient.DeleteShelfAsync(name);
-           // FIXME: inspect the results
-       }
-       // [END sample_core]
+       LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
+       ShelfName name = new ShelfName("[SHELF_ID]");
+       await libraryServiceClient.DeleteShelfAsync(name);
+       // FIXME: inspect the results
+   }
+   // [END sample_core]
    }
 
    // [END sample]
-   public static void Main(String[] args)
+   public static Task Main(string[] args)
```

Comment:
Sorry, this still needs an `async` before the `Task`:
`public static async Task Main(string[] args)`